sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -909,13 +908,8 @@ synchronized void completeWorkflow() {
         mUpdateDialog = null;
         mUpdateSetupFailedDialog = null;
         mUnknownSourcesDialog = null;
-        if (mReleaseDownloader != null) {
-            mReleaseDownloader.delete();
-            mReleaseDownloader = null;
-        }
         if (mReleaseDownloaderListener != null) {
             mReleaseDownloaderListener.hideProgressDialog();
-            mReleaseDownloaderListener = null;
```

Comment:
I don't understand why this is not cleaned anymore. Can you explain?

Reply:
It's revert of original logic. We don't clean loader and listener here - because we still need the downloaded file to install. Clearing downloader deletes the file. Cleaning listener without downloaded doesn't look right

Reply:
I was looking at outdated code, the latest has the `= null`